docker-entrypoint.sh

```diff
@@ -43,6 +43,14 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 			EOF
 			chown www-data:www-data .htaccess
 		fi
+	else
+		CONTAINER_WORDPRESS_VERSION=$(php -r "include('/usr/src/wordpress/wp-includes/version.php'); echo \$wp_version;")
+		INSTALLED_WORDPRESS_VERSION=$(php -r "include('wp-includes/version.php'); echo \$wp_version;")
+		if ! [ "$CONTAINER_WORDPRESS_VERSION" == "$INSTALLED_WORDPRESS_VERSION" ]; then
```

Comment:
All the files are correctly owned by `www-data`, since the `tar cf - --one-file-system -C /usr/src/wordpress . | tar xf -` preserves the user id from `/usr/src/wordpress`.

```console
$ docker exec -it word bash
root@79cc11d29c37:/var/www/html# ls -l
total 188
-rw-r--r--  1 www-data www-data   418 Sep 25  2013 index.php
-rw-r--r--  1 www-data www-data 19935 Mar  5  2016 license.txt
-rw-r--r--  1 www-data www-data  7344 Aug 16 20:39 readme.html
-rw-r--r--  1 www-data www-data  5456 May 24 21:02 wp-activate.php
drwxr-xr-x  9 www-data www-data  4096 Sep 20 20:34 wp-admin
-rw-r--r--  1 www-data www-data   364 Dec 19  2015 wp-blog-header.php
-rw-r--r--  1 www-data www-data  1477 May 23 16:44 wp-comments-post.php
-rw-r--r--  1 www-data www-data  2853 Sep 26 18:31 wp-config-sample.php
-rw-r--r--  1 www-data www-data  3235 Sep 26 18:31 wp-config.php
drwxr-xr-x  4 www-data www-data  4096 Sep 20 20:34 wp-content
-rw-r--r--  1 www-data www-data  3286 May 24  2015 wp-cron.php
drwxr-xr-x 17 www-data www-data 12288 Sep 20 20:34 wp-includes
-rw-r--r--  1 www-data www-data  2382 May 23 16:44 wp-links-opml.php
-rw-r--r--  1 www-data www-data  3353 Apr 14 17:53 wp-load.php
-rw-r--r--  1 www-data www-data 34057 Jun 14 21:51 wp-login.php
-rw-r--r--  1 www-data www-data  7786 Jul 13 12:37 wp-mail.php
-rw-r--r--  1 www-data www-data 13920 Aug 13 16:02 wp-settings.php
-rw-r--r--  1 www-data www-data 29890 May 24 20:44 wp-signup.php
-rw-r--r--  1 www-data www-data  4035 Nov 30  2014 wp-trackback.php
-rw-r--r--  1 www-data www-data  3064 Jul  6 12:40 xmlrpc.php
```